.github/workflows/bumpup.yml

```diff
@@ -0,0 +1,51 @@
+name: Bump up gem version
+
+on:
+  workflow_dispatch:
+    inputs:
+      gem-version:
+        description: Gem version
+        required: true
+        type: string
+      email:
+        description: Your email
+        required: true
+        type: string
+
+jobs:
+  bump_up_gem_version:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+          fetch-depth: 0
+
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: '3.1'
+        bundler-cache: true
+
+    - name: config
+      run: |
+        git config --global user.email ${{ github.event.inputs.email }}
+        git config --global user.name ${{ github.actor }}
+
+    - name: update sunmodule
+      run: git submodule foreach git pull origin master
+
+    - name: bundle install
+      run: bundle install
```

Comment:
This step must be executed after update gem version.
Because this step updates the `Gemfile.lock`.